Pipfile

```diff
@@ -0,0 +1,32 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+flake8-bandit = "*"
+flake8-bugbear = "*"
+flake8-import-order = "*"
+flake8-string-format = "*"
+flake8-tidy-imports = "*"
+mccabe = "*"
+pep8-naming = "*"
+coverage = "*"
+unittest-xml-reporting = "*"
+
+[packages]
+django = ">=2.2"
+django-crispy-forms = "*"
+django-environ = "*"
+django-filter = "*"
+django-hosts = "*"
+djangorestframework = "*"
+djangorestframework-bulk = "*"
+uwsgi = "*"
+psycopg2-binary = "*"
+django-simple-bulma = "*"
+django-crispy-bulma = "*"
```

Comment:
The Poetry lock file had these pinned to prevent breaking changes from being suggested as packages available for upgrade. The Pipfile should do the same, for instance `django` should be specified like:
```sh
django = ">=2.2,<3.0"
```
if that's valid Pipenv syntax. Maybe it supports a pin operator like Poetry (https://poetry.eustace.io/docs/basic-usage/#version-constraints)?